packages/admin/cms-admin/src/generator/future/generateForm.ts

```diff
@@ -99,7 +102,9 @@ export function generateForm(
         gqlIntrospection,
         baseOutputFilename,
         fields: config.fields,
+        fragmentName,
```

Comment:
I would name that formFragmentName, or you might also use formConfig.fragmentName? (that doesn't have the default value though)

Reply:
I don't want to handle the default value multiple times so I renamed the variable to better match it's content.